routers/root.py

```diff
@@ -735,6 +735,7 @@ def page_wrapper(request: Request, className=""):
         with gui.div(id="main-content", className="container-xxl " + className):
             yield
 
+        context["current_year"] = datetime.datetime.now().year
```

Comment:
let's add this here? https://github.com/GooeyAI/gooey-server/blob/092768839c98def606a83194b8b15b57c4f2ddd8/routers/root.py#L710-L714

easier of keep track of what `context` has when all keys are set together

Reply:
@nikochiko made changes